content/en/docs/concepts/storage/volumes.md

```diff
@@ -543,6 +543,69 @@ spec:
       type: FileOrCreate
 ```
 
+### image
+
+{{< feature-state feature_gate_name="ImageVolume" >}}
+
+An `image` volume source represents an OCI object (a container image or
+artifact) which is available on the kubelet's host machine.
+
+One example to use the `image` volume source is:
+
+{{% code_sample file="pods/image-volumes.yaml" %}}
+
+The volume is resolved at pod startup depending on which `pullPolicy` value is
+provided:
+
+- `Always`: the kubelet always attempts to pull the reference. If the pull fails,
+  the kubelet sets the Pod to `Failed`.
+- `Never`: the kubelet never pulls the reference and only uses a local image or
+  artifact. The Pod becomes `Failed` if any layers of the image aren't already present
+  locally, or if the manifest for that image isn't already cached.
+- `IfNotPresent`: the kubelet pulls if the reference isn't already present on
+  disk. The Pod becomes `Failed` if the reference isn't present and the pull
+  fails.
```

Comment:
Try this (no shortcode needed), which should render as a description list. I didn't test the suggestion.

```suggestion
`Always`
: the kubelet always attempts to pull the reference. If the pull fails, the kubelet sets the Pod to `Failed`.

`Never`
: the kubelet never pulls the reference and only uses a local image or artifact. The Pod becomes `Failed` if any layers of the image aren't already present locally, or if the manifest for that image isn't already cached.

`IfNotPresent`
: the kubelet pulls if the reference isn't already present on disk. The Pod becomes `Failed` if the reference isn't present and the pull fails.
```